cmd/tools/sql/main.go

```diff
@@ -34,5 +34,7 @@ import (
 )
 
 func main() {
-	_ = sql.RunTool(os.Args)
+	if err := sql.RunTool(os.Args); err != nil {
+		os.Exit(1)
```

Comment:
Please log this error so we know what went wrong

Reply:
(In all three tools)

Reply:
the cli package already logs it like this:

```
 % ./temporal-sql-tool --ca tx_isolation='READ-COMMITTED' --ep myhost.com -p 3306 -u root -pw a_password create --plugin mysql --db temporal
Incorrect Usage: flag provided but not defined: -plugin

NAME:
   temporal-sql-tool create-database - creates a database

USAGE:
   temporal-sql-tool create-database [command options] [arguments...]

OPTIONS:
   --defaultdb value  optional default db to connect to, this is not the db to be created
```

Reply:
looking at the code for non-nil `app.Run` errors,
- `normalizeFlags` err is printed, then help text, then err returned
- usage err is printed, then help text, then err returned
- if required flags are missing, help text is printed then err returned
- `HandleExitCoder` prints other errors and exits with exit code
so i think we're good on the logging front

Reply:
Weird. Works for me

Reply:
I'm surprised `app.Run` doesn't handle exiting for us then, maybe that's where it should live.